### DISCUSSION TITLE
BPI R64

### DISCUSSION THREAD

**Discussion by fighterslife** [Ysurac/openmptcprouter#2163] [General]
hi
i'm new to the forum even though i've been following and texting your iso for a long time.
I bought a bpir64 v1.1 I tested all the iso present but the problem is always the same only the yellow LEDs light up
and there is no way to connect to the bpi can you help me ???

**Comment by Kirball** [comment]
I recently got a a banana pi r64 myself and cant' for the life of me figure out how to even get it start the software.  Were you ever able to figure things out and if so any advice you can give me because there is something more than just flashing the sd card that I should be doing and I can't figure it out.

**Comment by Kirball** [comment]
I got a serial cable and from what I can tell it's getting stuck in a boot loop.  There is something wrong with the image file so I'm assuming until someone writes a new version or someone with the ability to find out what is broken with this image file goes in and corrects it it's unusable on a BPi R64.

**Comment by Ysurac** [comment]
I'm working on it.

**Reply by Kirball** [reply]
Awesome, thanks so much!

**Reply by Kirball** [reply]
Any updates on if there is a working version yet?  If I can do anything to help with getting things working please let me know.
